code/game/objects/items/weapons/ipc_scanner.dm

```diff
@@ -0,0 +1,33 @@
+/obj/item/ipc_tag_scanner
```

Comment:
It seems redundant to add a new analyzer if we already have the robot analyzer.
The functionality of the IPC tag scanner should also be present in the robot analyzer.
(Possibly slower, ... but still present)

Reply:
They serve different functions, one tool that does everything is against the team's wishes.

Reply:
If the tag scanner is available at roundstart in robotics, that can be waived.
However I would still prefer basic functionality in the robot analyzer (as thats basically the robot equivilant to the health scanner)